### DISCUSSION TITLE
The callback when task success by using the task_id in websocket comsumers.py environment

### DISCUSSION THREAD

**Discussion by honghh2018** [celery/celery#8218] [Q&A]
# Checklist

- [x] I have checked the [issues list](https://github.com/celery/celery/issues?utf8=%E2%9C%93&q=is%3Aissue+label%3A%22Issue+Type%3A+Feature+Request%22+)
  for similar or identical feature requests.
- [ ] I have checked the [pull requests list](https://github.com/celery/celery/pulls?utf8=%E2%9C%93&q=is%3Apr+label%3A%22PR+Type%3A+Feature%22+)
  for existing proposed implementations of this feature.
- [ ] I have checked the [commit log](https://github.com/celery/celery/commits/main)
  to find out if the same feature was already implemented in the
  main branch.
- [ ] I have included all related issues and possible duplicate issues
  in this issue (If there are none, check this box anyway).

#### Related Issues

- I want to using the return task_id to moniter whether the task was success or not in difference environment not the task function revoked in django view.
For example:
the celery task revoked in django view, task=task1.delay(),
and  then i want to consumers.py of websocket environment to moniter the task1 status, when it success, i want to trigger a function to send a message to frontend html.  

#### Possible Duplicates

- None

# Brief Summary

# Design

## Architectural Considerations

None

## Diagrams

N/A

## Alternatives

None

[a comment by an automated account is not shown]

**Comment by honghh2018** [comment]
您好！已经收到邮件，祝好！                                    洪汉辉

**Comment by auvipy** [comment]
This is more of a tech support question then feature request
